Add Architecture, Functionality, Landing, dependencies.

martinalturria opened the pull request:
Add the necessary dependencies to use images, fonts, Auth0 and add first screen.

In each commit explain what is added

martinalturria committed:
I add the main skeleton of Flutter

Start the Flutter project for the Silver Express app

martinalturria committed:
Add files .gitattributes and .gitignore

martinalturria committed:
Merge branch 'main' into develop/martin

martinalturria committed:
Add dependencies and sources to the pub.

modify the pubspec file to add font types, the following dependencies are also added:
-auth0_flutter: 1.2.0
-google_fonts: ^4.0.3
-http: ^0.13.4
-flutter_custom_clippers: ^2.0.0
-flutter_vector_icons: ^1.0.0

martinalturria committed:
add folder image and assets/font.

Add folders to handle the different types of media for the project

martinalturria committed:
Add Landing view and main.dart functionality

modify the main.dart file, add my_app.dart and LandingScreen.

martinalturria committed:
modifies README.MD with instruction to compile in web.

roquemolina reviewed (dismissed):
That seems fine to me. I think with this we can start to code in the folders without interfering with each other. So far my account is being used for auth0. For the next PR it would be necessary for Silver to open an account in auth0 so that it is ready and the different profiles can be managed. (Jean: I still don't understand the profile system when logging in, if you could explain a bit that would be useful).

martinalturria committed:
Add Screen of the main page and Widgets for dropdown menu

I add the main screen by making the drop-down menu conditioning it by whether it is web or mobile with the screen size.

brunofoster03 reviewed (approved):
Correct
